client/main.lua

```diff
@@ -201,6 +201,12 @@ RegisterNetEvent('lockpicks:UseLockpick', function(isAdvanced)
     LockpickDoor(isAdvanced)
 end)
 
+---Sync Vehicle Lock to all players
+RegisterNetEvent('vehicle:client:locksync')
+AddEventHandler('vehicle:client:locksync', function(veh,status)
```

Comment:
Remove the addeventhandler, you only have to use RsgisterNetEvent and pass the function to it